content/articles/2017-08-googledrive-initial-release.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+title: googledrive initial release
+author: Jenny Bryan
+date: '2017-08-27'
+slug: googledrive-initial-release
+description: >
+  The first release on googledrive is now on CRAN. Operate on Google Drive files from R.
+categories:
+  - packages
+photo:
+  url: https://unsplash.com/photos/wgLPy2YBXuc
+  author: Paul Csogi
+tags: []
+---
+
+```{r setup, include = FALSE}
+library("googledrive")
+library("tidyverse")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+token_path <- "~/rrr/googledrive/tidyverse-noncaching-token.rds"
+googledrive::drive_auth(token_path)
+```
+
+```{r eval = FALSE, include = FALSE}
+library(magick)
+img <- image_read("content/articles/paul-csogi-18594.jpg")
+img_sq <- img %>% 
+  image_crop("2988x2988+1200+0") %>% 
+  image_scale("300x300")
+img_sq %>% 
+  image_write("content/articles/googledrive-initial-release-sq.jpg")
+img_wd <- img %>% 
+  image_crop("5000x1000+200+1400") %>% 
+  image_scale("1000x200")
+img_wd %>% 
+  image_write("content/articles/googledrive-initial-release-wd.jpg")
+```
+
+We are tickled pink to announce the initial CRAN release of the googledrive package. This is a collaboration between Jenny Bryan and tidyverse intern, [Lucy D'Agostino McGowan](http://lucymcgowan.com/) (blog post on that coming soon!).
+
+googledrive lets you deal with files on Google Drive from R. You can find, list, create, trash, delete, rename, move, copy, browse, download, share and publish them. Install it with:
```

Comment:
First sentence feels a bit awkward

Reply:
Are you talking about "tickled pink" sentence or "googledrive lets you deal with files on Google Drive from R"?

To be clear, the "tickled pink" phrase is a joke (it's not "pleased"!) but we could possibly keep it.

Reply:
"googledrive lets you ..."

(I like tickled pink!)

Reply:
I moved meatier sentences up and made the awkward sentence disappear.